### ISSUE TITLE
FIR is broken

### ISSUE THREAD

**Issue by algesten** [algesten/str0m#34]
Chrome complains like so:

```
[41956:36355:0116/203007.888959:WARNING:fir.cc(59)] Packet is too small to be a valid FIR packet.
[41956:36355:0116/203007.888998:WARNING:rtcp_receiver.cc(557)] 1 RTCP blocks were skipped due to being malformed or of unrecognized/unsupported type, during the past 10 second period.
```

**Comment by algesten** [comment]
Closed by 69de0ea

**Comment by algesten** [comment]
Closed by 375f095
